specification/indices/create_data_stream/IndicesCreateDataStreamRequest.ts

```diff
@@ -21,12 +21,23 @@ import { RequestBase } from '@_types/Base'
 import { DataStreamName } from '@_types/common'
 
 /**
+ * Creates a data stream.
+ * You must have a matching index template with data stream enabled.
  * @rest_spec_name indices.create_data_stream
  * @availability stack since=7.9.0 stability=stable
  * @availability serverless stability=stable visibility=public
+ * @index_privileges create_index
  */
 export interface Request extends RequestBase {
   path_parts: {
+    /**
+     * Name of the data stream, which must meet the following criteria:
+     * Lowercase only
+     * Cannot include `\`, `/`, `*`, `?`, `"`, `<`, `>`, `|`, `,`, `#`, `:`, or a space character
+     * Cannot start with `-`, `_`, `+`, or `.ds-`
+     * Cannot be `.` or `..`
+     * Cannot be longer than 255 bytes. Multi-byte characters count towards this limit faster.
+     */
```

Comment:
Just out of curiosity: will this render OK in the final docs, without making a list or adding punctuation?

Reply:
You're right, I haven't checked to see whether bulleted lists actually work. I usually just change them into a series of phrases separated by semi-colons. I'll change that now.